features/cloud/file.include/etc/sysctl.d/20-cloud.conf

```diff
@@ -55,6 +55,3 @@ net.ipv4.conf.all.send_redirects = 1
 
 # Promote a secondary IP address in case a primary IP address is removed
 net.ipv4.conf.all.promote_secondaries = 1
-
-# Forward Packets between interfaces.
-net.ipv4.ip_forward = 1
```

Comment:
No replacement for this directive?

Reply:
No, in line 47 of the same file
```
net.ipv4.conf.all.forwarding = 1
```
is already set.

Reply:
Next time, I will click on "Show all lines" before commenting. ;-)